tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -1,31 +1,55 @@
-import {
-    currentURL,
-    visit,
-} from '@ember/test-helpers';
+import { currentURL, fillIn, visit } from '@ember/test-helpers';
+
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
 import { ModelInstance } from 'ember-cli-mirage';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 import { t } from 'ember-intl/test-support';
 import { percySnapshot } from 'ember-percy';
+import { selectChoose } from 'ember-power-select/test-support';
 import { setBreakpoint } from 'ember-responsive/test-support';
 import { TestContext } from 'ember-test-helpers';
+
 import { module, test } from 'qunit';
 
-import { click, setupOSFApplicationTest } from 'ember-osf-web/tests/helpers';
 import FileModel from 'ember-osf-web/models/file';
+import NodeModel from 'ember-osf-web/models/node';
+import { Permission } from 'ember-osf-web/models/osf-model';
 import RegistrationModel from 'ember-osf-web/models/registration';
+import User from 'ember-osf-web/models/user';
+import { click, setupOSFApplicationTest } from 'ember-osf-web/tests/helpers';
 
 interface ThisTestContext extends TestContext {
-    registration: ModelInstance<RegistrationModel>;
+    component: ModelInstance<NodeModel>;
+    componentFile: ModelInstance<FileModel>;
+    currentUser: ModelInstance<User>;
     file: ModelInstance<FileModel>;
+    node: ModelInstance<NodeModel>;
+    projectFile: ModelInstance<FileModel>;
+    registration: ModelInstance<RegistrationModel>;
 }
 
 module('Acceptance | guid file | registration files', hooks => {
     setupOSFApplicationTest(hooks);
     setupMirage(hooks);
 
     hooks.beforeEach(function(this: ThisTestContext) {
-        this.registration = server.create('registration');
-        this.file = server.create('file', { target: this.registration, name: 'Test File' });
+        this.node = server.create('node');
```

Comment:
Instead of adding the node, component, and node and component files here, I recommend instead moving those to the tests that need them. That way you're not making all of these things for every test, just the ones that require it. Should speed up test execution a bit.

Reply:
Ok, let me re-test. I had broken it into individual tests previously and it had broken. I am testing again with the node, component, and their files at the individual test level.

Reply:
This is now passing.